compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -46,8 +47,29 @@ import org.jetbrains.skiko.SkiaLayerAnalytics
 class ComposePanel @ExperimentalComposeUiApi constructor(
     private val skiaLayerAnalytics: SkiaLayerAnalytics,
 ) : JLayeredPane() {
+    private var reducePresentationLatency: Boolean? = null
+
     constructor() : this(SkiaLayerAnalytics.Empty)
 
+    /**
+     * Creates a ComposePanel with the specified forceReducedLatency flag.
+     *
+     * @param reducePresentationLatency If true, the ComposePanel will try to reduce presentation latency when rendering.
+     * This flag is a hint and may not be respected by the underlying implementation.
+     * The reduction of latency may come at the cost of visual artifacts like screen tearing.
+     *
+     * If not set, the parameter from [SkikoProperties] will be used.
+     * @see [SkikoProperties.vsyncEnabled]
+     *
+     * @param skiaLayerAnalytics Analytics that helps to know more about SkiaLayer behaviour.
+     */
+    constructor(
+        reducePresentationLatency: Boolean,
```

Comment:
Let's add it as a last parameter to main constructor (marked as `@ExperimentalComposeUiApi`)